packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -149,7 +137,8 @@ describe('src/cy/commands/assertions', () => {
         expect(testCommands()).to.containSubset([
           { name: 'wrap', snapshots: 1, retries: 0 },
           { name: 'then', snapshots: 0, retries: 0 },
-          { name: 'wrap', snapshots: 2, retries: (r) => r > 1 },
+          { name: 'wrap', snapshots: 1, retries: 0 },
+          { name: 'should', snapshots: 1, retries: (r) => r > 1 },
```

Comment:
The assertion retries and snapshots no longer "belong" to the previous command, since .should() is now a full-fledged member of the command queue.

Reply:
How come both commands don't have 2 snapshots? if wrap was retried, shouldn't there be more than one snapshot?